Packs/TrendMicroDDA/Integrations/integration-TrendMicroDDA.yml

```diff
@@ -436,6 +452,8 @@ script:
                 hr['Detection Name'] = file_analyze_report['VirusName']['#text']
             #set the filename
             filename = hr['Detection Name'] if ('Detection Name' in hr) else file_analyze_report['FileSHA1']
+            if '.' not in filename:
+                filename = filename + ".txt"
```

Comment:
I think it is not needed, as you set in in 533 and do not use the value until there

Reply:
thought so too, but it's an if/else so two cases